test/resources/selinux.py

```diff
@@ -74,6 +78,10 @@
 
 
 def get_expected_ocp_microshift_fcontext_list() -> List[str]:
+    stdout = remote_sudo("bash -c 'source /etc/os-release && echo $VERSION_ID'")
+    if stdout == "9.3":
```

Comment:
Is this specific to 9.3 version, or it's 9.3 and above?

Reply:
I don't know, 9.4 might have even more (or different) entries

Reply:
Should we better add `stdout >= 9.3` to make it potentially future proof?